composer.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "detailnet/denner-client",
+    "description": "API Client for Denner Portal 2.0 Web Services",
+    "type": "library",
+    "keywords": [
+        "denner",
+        "detailnet",
+        "api-client"
+    ],
+    "homepage": "https://github.com/detailnet/denner-client",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Ivan Wolf",
+            "email": "ivan.wolf@detailnet.ch"
+        },
+        {
+            "name": "Curzio Della Santa",
+            "email": "curzio.della.santa@detailnet.ch"
+        }
+    ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@bitbucket.org:detailnet/denner-common.git"
+        }
+    ],
+    "require": {
+        "php": ">=5.5.0",
+        "guzzlehttp/guzzle": "~5.0",
+        "guzzlehttp/guzzle-services": "~0.5.0",
+        "detailnet/dfw-normalization": "~0.1.6",
```

Comment:
move to require-dev and add then the suggestion "For usage of the denner model response classes" for it, 

same for "zendframework/zend-paginator" above.